PowerEditor/src/lastRecentFileList.h

```diff
@@ -44,6 +44,8 @@ class LastRecentFileList
 public:
 	LastRecentFileList() {
 		_userMax = (NppParameters::getInstance())->getNbMaxRecentFile();
+		for (int i = 0; i < NB_MAX_LRF_FILE; i++)
+			_idFreeArray[i] = false;
 	};

```

Comment:
Yes, appveyor build is failing. So reverting the changes.

>c:\projects\notepad-plus-plus\powereditor\src\lastRecentFileList.h(108): error C2536: 'LastRecentFileList::LastRecentFileList::_idFreeArray' : cannot specify explicit initializer for arrays (..\src\ScitillaComponent\Buffer.cpp) [C:\projects\notepad-plus-plus\PowerEditor\visual.net\notepadPlus.vcxproj]
          c:\projects\notepad-plus-plus\powereditor\src\lastRecentFileList.h(108) : see declaration of 'LastRecentFileList::_idFreeArray'
          c:\projects\notepad-plus-plus\powereditor\src\lastRecentFileList.h(108) : see declaration of 'LastRecentFileList::_idFreeArray'